config.go

```diff
@@ -359,6 +359,11 @@ func loadConfig() (*config, error) {
 			)
 		}
 	}
+	
+	// Check if config file exist
+        if _, err := os.Stat(configFilePath); os.IsNotExist(err) {
```

Comment:
A config file isn't required.

Reply:
Ok, what about now? If a user  supplied a config file from the command line and it does not exist than we should raise an exception